sdk/servicebus/azure-servicebus/azure/servicebus/_pyamqp/aio/_transport_async.py

```diff
@@ -246,6 +248,7 @@ def __init__(
         self.host, self.port = to_host_port(host, port)
 
         self.connect_timeout = connect_timeout
+        self.read_timeout = read_timeout or READ_TIMEOUT_INTERVAL #TODO: read timeout like sync transport
```

Comment:
We don't use read timeouts in async the same as sync - instead we wrap the read call in an asyncio.wait with a timeout and handle the failure there. So we shouldn't need to configure this on the socket.